crowsnest/utils.py

```
@@ -0,0 +1,148 @@
+        log_subprocess_output(process.stdout, info_log_func, info_log_pre)
+    )
+    stderr_task = asyncio.create_task(
+        log_subprocess_output(process.stderr, error_log_func, error_log_pre)
+    )
+
+    return process, stdout_task, stderr_task
+
+
+def execute_shell_command(command: str, strip: bool = True) -> str:
+    try:
+        output = subprocess.check_output(command, shell=True).decode("utf-8")
```

Ruff (0.14.14)

`subprocess` call with `shell=True` identified, security issue

(S602)